src/App.js

```diff
@@ -22,34 +23,45 @@ export function App () {
         FILE_VIEW: 1,
     };
 
-    const urlSearchParams = new URLSearchParams(window.location.search, "?");
+    const urlHashParams = new VerbatimURLParams(window.location.hash, "#");
```

Comment:
Since these are only used in init, can we move them in there?

Reply:
Done.